x/axelarnet/keeper/msg_server.go

```diff
@@ -189,28 +189,28 @@ func (s msgServer) ConfirmDeposit(c context.Context, req *types.ConfirmDepositRe
 		return nil, fmt.Errorf("recipient chain '%s' is not activated", recipient.Chain.Name)
 	}
 
-	coin, err := NewCoin(ctx, s.ibcK, s.nexus, amount)
+	normalizedCoin, err := NewCoin(ctx, s.ibcK, s.nexus, coin)
```

Comment:
Just to confirm, this is just a renaming right?

Reply:
yes